programming/encoders.md

```diff
@@ -0,0 +1,60 @@
+# Encoders
+
+**Encoders measure the rotation of motors and axles.**
+
+We primarily use encoders to make our robot drive a specific distance in **autonomous**. The encoder tells us the angle that a motor is at, and from that we can calculate the distance that the robot has driven. We also can use encoders in **swerve drive** to measure the angle each module is turned at which allows us to make sure we have all our wheels facing the right direction.
+
+There's lots of different examples of encoders at work all around us, such as:
+  - Elevators - Encoders in elevators tell the elevator's controller device its position so that it can properly align itself with the floor and open up the doors at the correct position.
+  - Cranes - Some large cranes use encoders in order to collect accurate rotation information to know where it is.
+
+**How do Encoders (generally) work?**
+
+![Encoder Diagram](http://encoder.com/core/files/encoder/uploads/images/Encoder-exploded-COLOR-v2.jpg)
```

Comment:
Do we have permission to use this image?

Reply:
I don't think we need their explicit permission seeing as the point of the wiki page is to educate which to my (admittedly) little knowledge on copyright qualifies as fair use.  I'll call their community service rep tomorrow via phone and if it happens to be that we can't use it I'll edit the page accordingly.